pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: apptentive_flutter
 description: Apptentive SDK for Flutter
-version: 6.2.2
+version: 6.7.0
```

Comment:
I intentionally didn't bump the version as I wasn't aware if it is part of the release process as we used to do with the SDK builder. But we can use this PR for release as well.

Reply:
I'll have to refresh my memory how everything works, but I'm hoping we can use the GitHub actions to release this time. In any case, it probably would be good to leave this at 6.2.2 so we can see the diff in the release.